addons/account_peppol/models/res_partner.py

```diff
@@ -28,12 +28,12 @@ class ResPartner(models.Model):
     available_peppol_edi_formats = fields.Json(compute='_compute_available_peppol_edi_formats')
     peppol_verification_state = fields.Selection(
         selection=[
-            ('not_verified', 'Not verified yet'),
-            ('not_valid', 'Not on Peppol'),  # does not exist on Peppol at all
-            ('not_valid_format', 'Cannot receive this format'),  # registered on Peppol but cannot receive the selected document type
-            ('valid', 'Valid'),
+            ('not_verified', 'Unchecked'),
+            ('not_valid', 'Partner is not on Peppol'),  # does not exist on Peppol at all
+            ('not_valid_format', 'Partner missing VAT or Company ID'),  # registered on Peppol but cannot receive the selected document type
```

Comment:
Huh, that doesn't make sense ?
This state just says that the guy IS on Peppol, but can't receive the format selected in `invoice_edi_format`.
We could add a state in stable tho, but it needs to be set accordingly.